klogr/klogr.go

```diff
@@ -134,6 +134,11 @@ func flatten(kvList ...interface{}) string {
 }
 
 func pretty(value interface{}) string {
+	if err, ok := value.(error); ok {
+		if _, ok := value.(json.Marshaler); !ok {
+			value = err.Error()
```

Comment:
is it better to just `return err.Error()` here?

Reply:
I did that at first, but then the result that is printed is unquoted, which seemed strange

```
"msg"="test"  "err"=whoops
```

Reply:
golang's `json.Marshal` would quote objects of type `string`.
so `whoops` becomes `\"whoops"`.
https://play.golang.org/p/nXBXJYStLL6

Reply:
gotcha! thanks!